src/StorageMover/StorageMover.Autorest/test/New-AzStorageMoverAzSmbFileShareEndpoint.Tests.ps1

```diff
@@ -5,7 +5,7 @@ if(($null -eq $TestName) -or ($TestName -contains 'New-AzStorageMoverSmbFileShar
       $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
   }
   . ($loadEnvPath)
-  $TestRecordingFile = Join-Path $PSScriptRoot 'New-AzStorageMoverSmbFileShareEndpoint.Recording.json'
+  $TestRecordingFile = Join-Path $PSScriptRoot 'New-AzStorageMoverAzSmbFileShareEndpoint.Recording.json'
```

Comment:
The test case name not changed in line #1

Reply:
Updated